package.json

```diff
@@ -38,7 +38,11 @@
     "@pnpm/logger": "^1.0.0",
     "@pnpm/npm-resolver": "^0.3.1",
     "@pnpm/tarball-fetcher": "^0.3.1",
+    "@types/got": "^7.1.6",
```

Comment:
type dependencies of prod dependencies should be also prod dependencies

Reply:
ok